server/webapp/WEB-INF/vm/shared/_header.vm

```diff
@@ -53,7 +53,20 @@
     #if(!$current_tab && $currentTab.name)
         #set($current_tab = $currentTab.name)
     #end
-<body>
+    #set ($showAnalyticsDashboard = ($supportsAnalyticsDashboard && $userHasAdministratorRights))
```

Comment:
Maybe we can move this in the VelocityContext itself?

Reply:
Done